app/models/role.rb

```diff
@@ -17,6 +18,14 @@ def self.add_admin(person)
     find_or_create_by(person_id: person.id, name: "admin")
   end
 
+  def self.moderator?(person)
+    exists?(person_id: person.id, name: "moderator")
```

Comment:
And then three specs would be awesome, `expect(Role.moderator?(admin_person)).to be_true`, `expect(Role.moderator?(moderator_person)).to be_true`, `expect(Role.moderator?(normal_person)).to be_false`